bedrock/mozorg/templates/mozorg/home/includes/mr1-promo-ryan.html

```diff
@@ -0,0 +1,58 @@
+    </div>
+
+    <div class="c-fxpromo-graphic">
+      <svg class="c-fxpromo-noodle c-fxpromo-noodle-pill" xmlns="http://www.w3.org/2000/svg" width="246" height="648" viewBox="0 0 246 648">
+        <linearGradient id="green-yellow" x1="10" x2="250" y1="600" y2="10" gradientUnits="userSpaceOnUse">
+          <animateTransform attributeName="gradientTransform" type="rotate" from="360 50 20" to="0 20 400" dur="30s" repeatCount="indefinite"/>
+          <stop offset=".15" stop-color="#54ffbd"/>
+          <stop offset=".75" stop-color="#fff44f"/>
+        </linearGradient>
+        <path fill="url(#green-yellow)" d="M0 123v402.5c0 67.9 55.1 123 123 123s123-55.1 123-123V123C246 55.1 190.9 0 123 0S0 55.1 0 123z"/>
+      </svg>
+
+      <svg class="c-fxpromo-noodle c-fxpromo-noodle-spot" xmlns="http://www.w3.org/2000/svg" width="230" height="230" viewBox="0 0 230 230">
+        <linearGradient id="purple-blue" x1="30" x2="180" y1="0" y2="100" gradientUnits="userSpaceOnUse">
+          <animateTransform attributeName="gradientTransform" type="rotate" to="0 60 120" from="360 80 10" dur="30s" repeatCount="indefinite"/>
+          <stop offset="0" stop-color="#b833e1"/>
+          <stop offset="1" stop-color="#0060df"/>
+        </linearGradient>
+        <path fill="url(#purple-blue)" d="M115 230c63.5 0 115-51.5 115-115S178.5 0 115 0 0 51.5 0 115s51.5 115 115 115z"/>
+      </svg>
+
+      <svg class="c-fxpromo-noodle c-fxpromo-noodle-hollow-pill" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="203" height="536" viewBox="0 0 203 536">
+        <defs>
+          <linearGradient id="blue-purple" x1="-858.08" y1="808.16" x2="-857.15" y2="808.16" gradientUnits="userSpaceOnUse">
+            <animateTransform attributeName="gradientTransform" type="rotate" to="360 50 20" from="0 20 400" dur="30s" repeatCount="indefinite"/>
+            <stop offset="0" stop-color="#0060df"/>
+            <stop offset="1" stop-color="#9059ff"/>
+          </linearGradient>
+        </defs>
+        <path fill="url(#blue-purple)" d="M101.62,535.81A101.73,101.73,0,0,1,0,434.19H0V101.62A101.62,101.62,0,0,1,101.62,0h0A101.61,101.61,0,0,1,203.24,101.62V434.2A101.73,101.73,0,0,1,101.62,535.81ZM101.62,3A98.62,98.62,0,0,0,3,101.62V434.19a98.62,98.62,0,0,0,197.24,0V101.62A98.62,98.62,0,0,0,101.62,3Z"/>
+      </svg>
+
+      <svg class="c-fxpromo-noodle c-fxpromo-noodle-circle" xmlns="http://www.w3.org/2000/svg" width="44" height="44" viewBox="0 0 44 44">
+        <circle cx="22" cy="22" r="20.5" fill="none" stroke="#3fe1B0" stroke-width="3">
+          <animate attributeName="stroke" values="#3fe1B0;#fff44f;#3fe1B0;" dur="35s" repeatCount="indefinite" />
+        </circle>
+      </svg>
+
+      {{ high_res_img('img/home/2021/ryan.png', {'alt': '', 'width': '570', 'height': '529', 'class': 'c-fxpromo-photo'}) }}
+
+      <p class="c-fxpromo-caption">
+        <span class="c-fxpromo-caption-name">RyanNicole –</span>
```

Comment:
No space between first and last name here. Just wanted to check that's intentional.

Reply:
That's intentional: https://www.msryannicole.com/home